dev_tools/containers/README.md

```diff
@@ -42,11 +42,13 @@ There is minor differences, but all of Ginkgo's recipes install the following
 packages:
 + GNU compilers
 + LLVM/Clang
++ Intel Compilers
```

Comment:
Are we always installing Intel compilers? I thought that this was just for some container.

Reply:
Yes, they are always installed actually. I even specifically added Intel 2017 into the CUDA 9.0 container so that all of them have an intel compiler, which simplifies using the container. The only thing is CUDA 9.0 is not tested with Intel, because we already test that same compiler version (Intel 2017) with both CUDA 9.1 and CUDA 9.2.

Reply:
Good to know, thanks!